docs/testing_using_schema.md

```diff
@@ -0,0 +1,15 @@
+# Testing your schema and collecting data
+
+First, make sure your syntax is in correct jsonld format. Test all files with `@content` from command line:
+
+```
+npm install -g jsonlint
+grep -r --exclude-dir=node_modules --exclude-dir=ui --exclude-dir=.github "@context" . \
+   | cut -d: -f1 | xargs -I fname jsonlint -q fname
+```
+
+Or test individual files on the [json linter website(`https://jsonlint.com/`).
+
+Then you can view your schema by pointing the reproschema user interface to the path to your protocol: `http://schema.repronim.org/ui/#/?url=PATH_TO_PROTOCOL_SCHEMA`
+
+For example: `https://schema.repronim.org/ui/#/?url=https://raw.githubusercontent.com/sensein/covid19/master/protocol/Covid19_schema`
```

Comment:
When going to this URL it seems that the UI is displaying the schema that was pointed to on top of the voice schema.

I suspect this will confuse people. :wink:

Reply:
I didn't get you? the protocol schema passed as a parameter to `url` is being displayed correctly. 
are you talking about the image on the landing page? I had put that by default for any protocol, need to make it exclusive to the voice pilot study only. but the content of the landing page and the assessments in the protocol are being rendered correct.

Reply:
ha my bad.

I did not realize that the covid protocol also included the voice task activities.